.github/workflows/storage-layout-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Assert storage layout changes
+
+on:
+  push:
+    branches: [develop, master]
+  pull_request:
+    branches: [develop, master]
+
+jobs:
+  assert:
+    runs-on: ubuntu-latest
+    name: Assert storage layout 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Setup node.js version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: 'yarn'
+
+      - name: Install yarn deps
+        run: yarn install --immutable
+      
+      - name: Check storage-layout
+        uses: lidofinance/storage-layout-action@v1
+        with:
+          mode: check
+          src-folder: ./contracts
+          ignore-folders: "{test_helpers,template,mocks}"
```

Comment:
Do we have a chance to ignore some non-upgradable contracts somehow?

Reply:
I think we can. We can specify names of the folders to ignore. With contract names it will a bit harder and I would rather not do it

Reply:
Folders don't work

Reply:
Than let's keep it as is